webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback } from 'react'
+import cx from 'classnames'
+import { useDispatch, useSelector } from 'react-redux'
+import { ComparisonPlot } from 'dvc/src/plots/webview/contract'
+import { ComparisonTableLoadingCell } from './ComparisonTableLoadingCell'
+import { ComparisonTableMissingCell } from './ComparisonTableMissingCell'
+import styles from '../styles.module.scss'
+import {
+  changeDisabledDragIds,
+  setMultiPlotValue
+} from '../comparisonTableSlice'
+import { PlotsState } from '../../../store'
+import { zoomPlot } from '../../../util/messages'
+
+export const ComparisonTableMultiCell: React.FC<{
+  path: string
+  plot: ComparisonPlot
+}> = ({ path, plot }) => {
+  const multiValues = useSelector(
+    (state: PlotsState) => state.comparison.multiPlotValues
+  )
+  const dispatch = useDispatch()
+  const currentStep = multiValues[path] || 0
+
+  const { loading, url } = plot.imgs[currentStep]
+  const missing = !loading && !url
+
+  const addDisabled = useCallback(() => {
+    dispatch(changeDisabledDragIds([path]))
+  }, [dispatch, path])
+
+  const removeDisabled = useCallback(() => {
+    dispatch(changeDisabledDragIds([]))
+  }, [dispatch])
+
+  let imageContent = (
+    <button
+      className={cx(styles.imageWrapper, styles.multiImageWrapper)}
+      onClick={() => zoomPlot(url)}
+      data-testid="image-plot-button"
+    >
+      <img
```

Comment:
Things that I thought could be done in a followup:

* Currently we show the step based off the index of the array which wouldn't work correctly if the user deselected some of the images in the plots sideview tree. A better way to calculate the step could be relying on the image file name or getting the index when collecting the images on the backend. 
* Save the `multiValues` across sessions.

Reply:
> Currently we show the step based off the index of the array which wouldn't work correctly if the user deselected some of the images in the plots sideview tree. A better way to calculate the step could be relying on the image file name or getting the index when collecting the images on the backend.

Should we treat the whole thing as a single plot and not allowed selecting/de-selecting of steps?

> Save the multiValues across sessions.

Are the plot steps reset when an experiment runs, or is that handled?

Reply:
> Should we treat the whole thing as a single plot and not allowed selecting/de-selecting of steps?

Yes, we could do that! That would make image and image path collection easier! Apologies if that was your original idea in https://github.com/iterative/vscode-dvc/pull/4319#discussion_r1277116939, I misunderstood 😊

Reply:
It definitely wasn't the original idea but let's try and get rid of `isMulti` as a flag if possible.

Reply:
Adjusted the code to turn off selection/de-selection of steps. 

> Are the plot steps reset when an experiment runs, or is that handled?

Nope, the plot steps would only reset if you close the "session". They'll keep their current value during an experiment run :)